pkgs/development/python-modules/deezer-python/default.nix

```
@@ -0,0 +1,40 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pythonOlder
+, requests
+, tornado
+, poetry
+}:
+
+buildPythonPackage rec {
+  pname = "deezer-python";
+  version = "2.2.2";
+
+  src = fetchFromGitHub {
+    owner = "browniebroke";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1l8l4lxlqsj921gk1mxcilp11jx31addiyd9pk604aldgqma709y";
+  };
+
+  disabled = pythonOlder "3.6";
+
+  format = "pyproject";
+
+  nativeBuildInputs = [
+    poetry
```

`poetry` will work but if upstream allows to use [`poetry-core`](https://github.com/browniebroke/deezer-python/blob/main/pyproject.toml#L61) we should also only pull-in the build backend.

```suggestion
    poetry-core
```